ios/Classes/SwiftFlutterWebAuthPlugin.swift

```diff
@@ -46,8 +46,18 @@ public class SwiftFlutterWebAuthPlugin: NSObject, FlutterPlugin {
                 let session = ASWebAuthenticationSession(url: url, callbackURLScheme: callbackURLScheme, completionHandler: completionHandler)
 
                 if #available(iOS 13, *) {
-                    guard let provider = UIApplication.shared.delegate?.window??.rootViewController as? FlutterViewController else {
-                        result(FlutterError(code: "FAILED", message: "Failed to aquire root FlutterViewController" , details: nil))
+                    if var topController = UIApplication.shared.keyWindow?.rootViewController {
```

Comment:
There is still code that references the `provider` variable on line 55/65 so I don't think that these changes will compile 😅

Reply:
Yeah missed it when I merged master.